react-native-plugin.podspec

```diff
@@ -25,12 +25,12 @@ Pod::Spec.new do |s|
   s.author        = { "Bluedot Innovation" => "https://www.bluedot.io" }
   s.platform      = :ios, '10.0'
   s.swift_version = '5.0'
-  s.source       = { :git => "https://github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin.git" }
+  s.source       = { :git => "https://github.com/Bluedot-Innovation/Bluedot-React-Native-Plugin.git"}
 
   s.source_files  = "ios/**/*.{h,m,swift}"
   s.requires_arc  = true
 
-  s.dependency "BluedotPointSDK"
+  s.dependency "BluedotPointSDK", "15.6.4"
```

Comment:
explicitly specify PointSDK iOS version in podspec, otherwise the consumer app will always download latest public release of the SDK